checker/src/org/checkerframework/checker/i18nformatter/I18nFormatterTransfer.java

```diff
@@ -86,7 +87,8 @@ public I18nFormatterTransfer(I18nFormatterAnalysis analysis, I18nFormatterChecke
                 elseStore.insertValue(firstParam, builder.build());
 
             } catch (FlowExpressionParseException e) {
-                // errors are reported at declaration site
+                // report errors here
+                checker.report(e.getResult(), node);
```

Comment:
Shouldn't this be an i18nformat.invalid.formatfor error instead of flowexpr.parse.error?